py/utils.py

```diff
@@ -231,7 +231,7 @@ def openai_request(url, data, options):
 
 def print_info_message(msg):
     vim.command("redraw")
-    vim.command("normal \\<Esc>")
```

Comment:
As `<esc>` is not remapped in aichat buffers, I wonder why `normal!` is not used? What's the intended effect of allowing user overrides of `<esc>` in normal mode? Maybe that already helps your use case?